Neos.ContentRepository.Core/Classes/ContentRepository.php

```diff
@@ -141,18 +143,24 @@ public function handle(CommandInterface $command): void
      */
     public function projectionState(string $projectionStateClassName): ProjectionStateInterface
     {
+        if (!isset($this->projectionStateCache)) {
+            foreach ($this->projectionsAndCatchUpHooks->projections as $projection) {
+                if ($projection instanceof ContentGraphProjectionInterface) {
```

Comment:
So but that means you cannot get the CG projection here anymore? not sure that is good?!

Reply:
I think that's a _good_ thing (see comment above re PR privileges)